rules.disabled/Rejected_print_job.yaml

```diff
@@ -1,24 +1,22 @@
-name: No_uploads_in_10_mins
-description: 'Logging from the badgemanagement service suggests that photo uploads have stopped.'
+name: print_job_rejected_for_60_mins
 type: flatline
 index: filebeat-*
-threshold: 2
+threshold: 1
 use_count_query: true
 doc_type: doc
 timeframe:
-  minutes: 10
+  minutes: 60
 realert:
-  minutes: 5
+  minutes: 10
 exponential_realert:
   minutes: 60
 filter:
 - query:
     query_string:
-      query: "beat.name:badgemanagement AND message:\"photo upload successful\""
+      query: "bluebadge.message:\"\""
```

Comment:
mm? where's the query?

Reply:
good catch

Reply:
Ah, this is the one requirement that was dropped from the ticket. Also note 'rules.disabled' directory. There is currently no query that can resolve this in ELK

Reply:
Ah, perfect :)